test/perl/TimescaleNode.pm

```diff
@@ -56,4 +56,19 @@ sub psql_is
 		"$testname: psql output check");
 }
 
+# remove leading and trailing whitespace
+sub strip
+{
+  my ($self, $str) = @_;
+	$str =~ s/^\s+|\s+$//g;
+  return $str;
+}
```

Comment:
Nit: This is so simple that I'm not sure a function is needed, but whatever. Also, indentation makes it look like it is an `if`, and `$self` is never used, so maybe just skip it and not define it as part of the object (that is, just skip it entirely).